ledger/src/blockstore.rs

```diff
@@ -2188,6 +2188,11 @@ impl Blockstore {
         self.transaction_memos_cf.put(*signature, &memos)
     }
 
+    /// Returns the lock that protects lowest-cleanup-slot.  Holding this lock
+    /// will prevent the ledger cleanup service from cleaning slots.
```

Comment:
nit: For ease of searching across the codebase, I think it could be helpful to use the exact names, `lowest_cleanup_slot` and `LedgerCleanupService`.